debezium-core/src/main/java/io/debezium/relational/TableImpl.java

```diff
@@ -96,7 +96,8 @@ public String toString() {
         return sb.toString();
     }
 
-    protected void toString(StringBuilder sb, String prefix) {
+    @Override
+    public void toString(StringBuilder sb, String prefix) {
```

Comment:
This is just a technical method, I don't think this belongs to the interface. Is not this unnecessary code optimization?

Reply:
Indeed, let's not expose it on the interface. `toString()` shouldn't be invoked at runtime anyways, so shouldn't make any difference.

Reply:
This was to allow `Tables` to utilize `Table` over `TableImpl` (it utilizes the method for it's `toString())`. I can rework the method to utilize the normal toString or throw in casting instead.

Reply:
The latter sounds good.